styles/_icons.scss

```diff
@@ -0,0 +1,72 @@
+
+.icons-header{
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	gap: 1rem;
+	max-width: 1080px;
+	margin:0 auto;
+	padding:1rem;
+	* {
+		margin: 0;
+	}
+	input[type=text] {
+		width: 100%;
+		border: 1px solid var(--light);
+		padding: 1rem;
+		font-size: var(--step--2);
+		&:focus{
+			outline: 1px solid var(--highlight);
+		}
+	}
+}
+.icons-feature{
+	display: grid;
+	grid-template-columns: repeat(1, minmax(0, 1fr));
+	gap: 1rem;
+	max-width: 1080px;
+	margin:0 auto;
+
+	padding:1rem;
+	@media screen and (min-width: 768px) {
+		grid-template-columns: repeat(3, minmax(0, 1fr));
+	}
+	@media screen and (min-width: 1080px) {
+		grid-template-columns: repeat(5, minmax(0, 1fr));
```

Comment:
I think at most, we should have 4 columns, since some of the names are kind of long

Reply:
What about a smaller font size?